samples/graphene-hello/README.md

```diff
@@ -1,15 +1,15 @@
-# Graphene "Hello World!" sample
-This sample shows how to run a [Graphene](https://github.com/oscarlab/graphene) application in Marblerun.
+# Graphene "Hello World!" example
+This example shows how to run a [Graphene](https://github.com/oscarlab/graphene) application in Marblerun.
 
-In essence, you have to add the `premain` process either by manually adding the Marblerun premain process as the entry point for your application (see: `spawn`), or by automatically injecting it as a preloaded shared library via Graphene's LD_PRELOAD feature (see: `preload`).
+In essence, you have to add the `premain` process either by manually adding the Marblerun premain process as the entry point for your application (see: [`spawn`](spawn)), or by automatically injecting it as a preloaded shared library via Graphene's LD_PRELOAD feature (see: [`preload`](preload)).
 
-`premain` will contact the Coordinator, set up the environment, and run the actual application. See the commented hello.manifest.template in the corresponding folder for details.
+`premain` will contact the Coordinator, set up the environment, and run the actual application. See the commented `hello.manifest.template` in the corresponding folder for details.
```

Comment:
```suggestion
`premain` will contact the Coordinator, set up the environment, and run the actual application. See the commented [`hello.manifest.template`](hello.manifest.template) in the corresponding folder for details.
```

Hope this links works locally.

Reply:
This won't work, since the `graphene-hello` folder does not contain such a template. In the subfolders `preload` and `spawn`, where such a file exists, the link to the file already exists :wink:

Reply:
Oh, didn't see this was actually the README.md from the root sample directory. My bad, that's fine then!